app/components/SearchResults.tsx

```diff
@@ -3,6 +3,9 @@ import type JSXNode from "~/types/JSXNode"
 import { SearchResultEnum } from "~/routes/search/searchResultEnum"
 import SearchResultEntries from "./search/SearchResultEntries"
 import SearchResultMeanings from "./search/SearchResultMeanings"
+import SearchResultUsageNotes from "./search/SearchResultUsageNotes"
+import SearchResultCanadianism from "./search/SearchResultCanadianism"
+import SearchResultFistNotes from "./search/SearchResultFistNotes"
 
 interface SearchResultsProps {
   data: any[]
```

Comment:
Can we type this better? (Would something like `string[]` be appropriate?)

Reply:
unfortunately I don't think we can. We're getting data through `GetSearchResults` in the file `$text`, and this is what is being passed to the search result class. This is an any[] type because `GetSearchResults` returns the search results for Entries, meanings, etc.

Reply:
OK for now.